files/logind_override.conf

```diff
@@ -0,0 +1,2 @@
+[Login]
```

Comment:
This is definitely a better way to handle the logind config change than blockreplacing!